opentutor_classifier/opentutor_classifier/config.py

```diff
@@ -8,3 +8,7 @@
 
 def confidence_threshold_default() -> float:
     return float(environ.get("CONFIDENCE_THRESHOLD_DEFAULT", "0.6"))
+
+
+def feature_generation_disabled() -> bool:
```

Comment:
understanding from last comment that we don't need this anymore? if that's the case, then just delete